### ISSUE TITLE
Accessbility (plus haptics) QA of HSP

### ISSUE THREAD

**Issue by brea11y** [department-of-veterans-affairs/va-mobile-app#8840]
Since we were unable to test the home screen personalization work with screen reader users, perform a thorough accessibility audit (including the new haptics implementation) of the mobile app, focusing primarily on home screen personalization. Verify that no other accessibility issues exist (that could be related to the HSP work).

**Comment by brea11y** [comment]
Android a11y + haptics QA complete. Noticed no accessibility issues and haptics worked as expected. Links, buttons, and errors read out as expected with TalkBack and I do not foresee any issues for screen reader users with the HSP updates.

Still having some issues getting access to TestFlight on my iOS test device. Binny said he could help shortly. Will complete QA as soon as I can get access to the build.

**Comment by brea11y** [comment]
iOS a11y + haptics QA complete. Noticed no accessibility issues and haptics worked as expected. Links, buttons, and errors read out as expected with VoiceOver and I do not foresee any issues for screen reader users with the HSP updates.
